### DISCUSSION TITLE
Is there a way to force the container to exit?

### DISCUSSION THREAD

**Discussion by derek-mba** [shinsenter/php#34] [Q&A] [answered]
I have a script in /startup that I would like to bring the container down if it fails. `s6-svscanctl` _looks_ like it should be the tool for the job, but I can't figure out how to use it.

Is there a simple solution?

**Comment by shinsenter** [comment, marked as answer]
Hi @derek-mba 

You simply need to run the command `kill 1` to exit the container.

**Reply by derek-mba** [reply]
Uh… d'oh!

While googling for a solution, I saw somebody say that it was specifically forbidden to `kill 1`, so I didn't try it. Obviously, that person was either wrong or talking about something completely different!

Thanks
